test/java/src/test/java/TestJsonSerializer.java

```diff
@@ -0,0 +1,120 @@
+import java.io.IOException;
+import java.io.StringWriter;
+import java.math.BigDecimal;
+import java.io.File;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashSet;
+
+import com.fasterxml.jackson.core.*;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.test.example.tuple.complex.BuyItemFormComplex;
```

Comment:
Rename package `com.test.example.tuple.complex` to `io.sysl.reljam.gen.tuple` or similar as we own the `sysl.io` domain.

Block first party, third-party and local imports - ie move all `io.sysl.reljam.gen.tuple` imports to the bottom.